backend/python-functions/realtime_rt.py

```diff
@@ -126,10 +126,6 @@ def compute_r_t(historical_case_counts):
         raise ValueError(
             'Case counts must be cumulative and monotonically increasing')
 
-    # we believe duplicates are mostly an artifact of little-to-no testing
-    # and should be excluded from the model
-    case_df = case_df.drop_duplicates(keep='first')
```

Comment:
Github won't let me comment on it directly but there is an error message below on L134 that makes reference to duplicate case counts; if that requirement is changing we should also change the message to avoid confusion. (I think saying "two days of data" instead of "two days of non-repeating case counts" is probably sufficient?)

Reply:
Good call out! I've updated the error message accordingly and changed the python test to make sure only the first day is dropped in the output.